open-defi-api.yaml

```diff
@@ -239,6 +239,17 @@ paths:
           required: false
           schema:
             type: string
+
+        - name: address
+          in: query
+          description: |
+            Search for DEX venues by token contract address
+          required: true
```

Comment:
Oops I should have made the comment in the PR, `required` should be set to `false` here as we don't want to force users to use it